training/refit/__init__.py

```diff
@@ -97,15 +100,32 @@ def save(self,
     def __get_file_path(self, object_name: str):
         return f"import/{self.project_guid}/{object_name}"
 
-    def import_file(self,
-                    file_path: str,
-                    object_name: str,
-                    delete_when_complete: bool = True, ) -> str:
+
+
+    def import_data(self,
+                dataframe: pd.DataFrame(),
+                object_name: str) -> str:
```

Comment:
We should not need to make an object name, can we dynamically create one using a GUID?